dags/process_elife_zip_dag.py

```diff
@@ -57,8 +58,16 @@ def get_expected_elife_article_name(file_name: str) -> str:
     return article_name.group() + '.xml'
 
 
+def get_article_from_previous_task(context: dict) -> ElementTree:
+    article_bytes = get_return_value_from_previous_task(context)
+    previous_task = get_previous_task_name(context)
+    message = 'Article bytes were not passed from task %s' % previous_task
+    assert article_bytes and isinstance(article_bytes, bytes), message
```

Comment:
wonder if we should introduce a stronger type, like a Value Object that wraps the bytes and can still be pickled?

Reply:
(more of a suggestion for later as this is getting quite big)